AUTHORS

```diff
@@ -0,0 +1,106 @@
+md678685 <mathew678@hotmail.co.uk>
+Adam Dobrawy <naczelnik@jawnosc.tk>
+Alex Bradley <abrad12122@gmail.com>
+Jonathan Pepler <jonpepler@Jons-MacBook-Pro.local>
+much <much@localhost.localdomain>
+pchauncey <pchauncey@gmail.com>
+Richard <richard.anaya@gmail.com>
+Who? Me?! <mark-i-m@users.noreply.github.com>
+ozyx <jessemazzella@gmail.com>
+ECrownofFire <ecrownoffire@gmail.com>
+J0hn- <jonathan.platteau@yahoo.fr>
+Gustavo Rodrigues <qgustavor@users.noreply.github.com>
+MRHwick <MatthewRHardwick@gmail.com>
+Unknown <dehodson@gmail.com>
+scaule <scaule@clever-age.com>
+Phil Rukin <philipp@rukin.me>
+golergka <golergka@Gmail.com>
+DasSkelett <DasSkelett@users.noreply.github.com>
+Ben Barth <benbarth@users.noreply.github.com>
+Joseph Hassell <josephh2018@gmail.com>
+eukaryote <minerguy31@gmail.com>
+Alex <alexisr245@gmail.com>
+Phil <phil.ficus@gmail.com>
+John <J0hn-@users.noreply.github.com>
+Konrad Borowski <xfix@protonmail.com>
+Tarun <tarun.batra00@gmail.com>
+Konrad Borowski <x.fix@o2.pl>
+Andrew Dassonville <dassonville.andrew@gmail.com>
+Matthew Hardwick <mrhwick@users.noreply.github.com>
+Patrick Chauncey <pchauncey@gmail.com>
+Patrick Nelson <patnelson@ebay.com>
+Peter Geiss <pgeiss@users.noreply.github.com>
+Quentin Gerodel <Swizz@users.noreply.github.com>
+Jeff Bowen <jblz@users.noreply.github.com>
+Smittyvb <smittyvb@gmail.com>
+Carson Radtke <8radtke@arrowheadschools.org>
+Tibor Nagy <xnagytibor@gmail.com>
+John <flibustier@users.noreply.github.com>
+Jon Paul Uritis <jonpaul.act@gmail.com>
+eamanu <emmanuelarias30@gmail.com>
+hawkfalcon <contact@hawkfalcon.com>
+jakejdavis <jakeyjdavis@gmail.com>
+Jon Paul Uritis <jonpaul@lojistic.com>
+Jon Pepler <jon1467@users.noreply.github.com>
+philipz <philipzheng@gmail.com>
+KnappeGEIL <KnappeGEIL@users.noreply.github.com>
+KnappeGEIL <fabian@bermel.org>
+Drew Hodson <dehodson@gmail.com>
+Leon Mok <mok.leon@gmail.com>
+Anthony Alves <Cvballa3g0@gmail.com>
+MINIMAN10000 <miniman10000@gmail.com>
+Anthony Nixon <ajn0592@gmail.com>
+Paul <paul.a.dibiase@gmail.com>
+Adrian Legaspi <aki.legaspi@gmail.com>
+Alois <aloisdegouvello@live.fr>
+Ameliorate <Ameliorate@users.noreply.github.com>
+Arturs Demiters <arturs@codecraft.lv>
+Benjamin Manns <benmanns@gmail.com>
+Bradley Monk <brad.monk@gmail.com>
+Campbell <The6P4C@users.noreply.github.com>
+Carson Radtke <carsonRadtke@users.noreply.github.com>
+Dave Hughes <DHughes@XumaK.com>
+ELI JOSEPH BRADLEY <EtherTyper@users.noreply.github.com>
+Ethan Madden <maddene@madden.ninja>
+Evan Hopper-Moore <evanhoppermoore@gmail.com>
+GRAND Léo <leograndcontact@gmail.com>
+Gökberk Yaltıraklı <webdosusb@gmail.com>
+Jacob <brokendwarf@gmail.com>
+James Watling <james@guavapass.com>
+Jeff Bowen <jeff@automattic.com>
+Jesse Mazzella <ozyx@users.noreply.github.com>
+Josh Grigonis <josh.grigonis@gmail.com>
+Kevin Godby <godbyk@gmail.com>
+Kevin Kielholz <kevinkielholz@web.de>
+Leigende <alexisr245@gmail.com>
+Léo <leograndcontact@gmail.com>
```

Comment:
And I (Im duplicated 3times, because I didnt configurate my git correctly & I'm using multiple computers)